pajbot/modules/subscriber_fetch.py

```diff
@@ -50,6 +50,9 @@ def _update_subscribers(self):
             else:
                 raise
 
+        # Dedupe the list of subscribers
+        subscriber_ids = list(set(subscriber_ids))
+
```

Comment:
could we put this deduplication into the API wrapper itself?

Reply:
Sure